src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -53,6 +53,21 @@ export interface WDSJob {
   updated: string;
 }
 
+// The source of truth of valid capabilities can be found in: https://github.com/DataBiosphere/terra-workspace-data-service/blob/main/service/src/main/resources/capabilities.json
+export type KnownCapability =
```

Comment:
Super nit: I suspect no other module will ever use `UnknownCapability`. It would be nice if the external interface for this enum was just `Capability` vs `KnownCapability`.